docs/source/design.rst

```diff
@@ -29,6 +29,12 @@ named arguments:
     :lines: 192-
     :lineno-start: 192
 
+We can go ahead and run this code from a file by using the ``mt`` commandline
+tool::
+  mt eval mafiabot.mt chat.freenode.net
```

Comment:
Last I tried, `mt eval` did not pass through its argv, so this doesn't work. Happy to be shown that I'm wrong.